### ISSUE TITLE
Disable viewport zoom on touch devices

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#719]
It shouldn't be possible to zoom the UI on touch devices. Currently it is possible which can lead to things like this:
![Simulator Screen Shot - iPhone 8 - 2021-03-25 at 11 26 09](https://user-images.githubusercontent.com/110222/112458286-ed240300-8d5c-11eb-9678-ec724ea0abfb.png)

**Comment by jacobwod** [comment]
Note to self: maybe this will do?

<meta name="viewport" content="width=device-width, user-scalable=no" />

**Comment by sweco-seqtob** [comment]
Seems like changing the meta-tag is not gonna work for iOS 10 + because Apple decided to override the behaviour

"To improve accessibility on websites in Safari, users can now pinch-to-zoom even when a website sets user-scalable=no in the viewport." - https://stackoverflow.com/questions/37808180/disable-viewport-zooming-ios-10-safari

Not sure if this still is the case but need to test it

**Comment by jacobwod** [comment]
My proposed solution didn't work. Neither did the one you told me to test. Which one was that? (So we can keep track of stuff that doesn't work.)

```
// no effect
<meta name="viewport" content="width=device-width, user-scalable=no" />
```

**Comment by sweco-seqtob** [comment]
@jacobwod Can ut add the other solutions you tried?

**Comment by jacobwod** [comment]
```
// index.html
    <style>
      html,
      body {
        touch-action: none;
        overflow: hidden;
      }
    </style>
```

**Comment by jacobwod** [comment]
I think I've found a solution, coming up later today.

**Comment by jacobwod** [comment]
Yeah it took some trial-and-error but hopefully I got it right in the end. Until they change something in future versions of Safari, that is. 😉
